### DISCUSSION TITLE
Icon request: icon-truck-trailer

### DISCUSSION THREAD

**Discussion by chrysler5798** [FortAwesome/Font-Awesome#21203] [Icon Requests]
Hello,
Following the release of trailer icon (fa-trailer, issue #7369) which is for car, I would like to suggest adding also a truck trailer.

**What can it be used for?**
It would be useful for transportation (logistics) field.

**Example image**
![Truck trailer](https://cdn0.iconfinder.com/data/icons/transport-by-truck/120/trailer_parked-512.png)
I took this image from @ShawnCZek's comment on #7369 issue.

**Comment by floranberthaud** [comment]
# **+1**

**Comment by sensibleworld** [comment]
This is a quick update to let folks know that we’re refining our icon request workflow behind the scenes, and as part of that we’re going through all open requests and filing them into several new buckets to be addressed.

In this case we’ve added this issue to our new list to be created! We can’t promise any firm timelines right now, but this will help us keep track of it.

We’ll update this issue when the icon is released into the wild 🙂
